operator/main.go

```diff
@@ -254,7 +254,10 @@ func runOperator() {
 				if identity == operatorID {
 					log.Info("Leading the operator HA deployment")
 				} else {
-					log.WithField("operator-id", operatorID).Infof("Operator with ID %q elected as new leader", identity)
+					log.WithFields(logrus.Fields{
+						"newLeader": operatorID,
+						"identity":  identity,
+					})
```

Comment:
I think we still need the `Info()` part with a message, otherwise we don't know what's happening and I'm not even sure a message is logged.

Reply:
Like this? 
```
log.WithFields(logrus.Fields{
	"newLeader": operatorID,
	"identity":  identity,
}).Info("Leader re-election Complete")
```
What should I put the message?

Reply:
Yep, I think that's good (without the upper case for `Complete`). Others reviewers may prefer a different message though; will see :smiley: